edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -117,7 +117,13 @@
   <section aria-hidden="true" class="modal staff-modal" id="{{ id }}-grade" style="height: 75%">
     <div class="inner-wrapper" style="color: black; overflow: auto;">
       <header><h2>{% trans "Staff Graded Assignment" %}</h2></header>
-      <br/>
+      <br>
+
+      <div class="grades_published">
+        <label for="grades-published">Grades published:</label>
+        <input id="grades-published" name="grades_published" type="checkbox" {% if grades_published %} checked {% endif %}>
+      </div>
```

Comment:
https://www.google.hr/search?client=firefox-b-ab&dcr=0&biw=1280&bih=615&tbm=isch&sa=1&ei=V2RgWoeVOMmlsgHFiYa4Ag&q=checkbox+labels&oq=checkbox+labels&gs_l=psy-ab.3..0i24k1.12613.14293.0.14956.9.6.1.2.2.0.175.783.1j5.6.0....0...1c.1.64.psy-ab..0.9.803...0j0i5i30k1j0i8i30k1j0i13k1.0.zqqrH_G6A_c#imgrc=tTFAUIslSjY1cM:


Notice the difference.